sapi/fuzzing/config.m4

```diff
@@ -0,0 +1,61 @@
+AC_MSG_CHECKING(for CLANG fuzzer SAPI)
+
+PHP_ARG_ENABLE(fuzzer,,
+[  --enable-fuzzer          Build PHP as clang fuzzing test module (for developers], no)
+
+dnl For newer clang versions see https://llvm.org/docs/LibFuzzer.html#fuzzer-usage
+dnl for relevant flags.
+
+dnl Macro to define fuzzing target
+dnl PHP_FUZZER_TARGET(name, target-var)
+dnl
+AC_DEFUN([PHP_FUZZER_TARGET], [
+  PHP_FUZZER_BINARIES="$PHP_FUZZER_BINARIES $SAPI_FUZZER_PATH/php-fuzz-$1"
+  PHP_SUBST($2)
+  PHP_ADD_SOURCES_X([sapi/fuzzer],[fuzzer-$1.c fuzzer-sapi.c],[],$2)
+])
+
+if test "$PHP_FUZZER" != "no"; then
+  PHP_REQUIRE_CXX()
+  PHP_ADD_MAKEFILE_FRAGMENT($abs_srcdir/sapi/fuzzer/Makefile.frag)
+  SAPI_FUZZER_PATH=sapi/fuzzer
+  PHP_SUBST(SAPI_FUZZER_PATH)
+  if test -z "$LIB_FUZZING_ENGINE"; then
```

Comment:
Where does this come from?

Reply:
This is Google's OSS-Fuzz, this is what is being used here: https://github.com/google/oss-fuzz/tree/master/projects/php. They are using similar infrastructure but a different library.